docs/Microsoft.Omex.Gating/GatingTutorial.md

```diff
@@ -0,0 +1,106 @@
+# How to use the Gating library
```

Comment:
Can you please add a paragraph explaining that in Omex Gating is used as a flighting framework for new service features. 

For example,  if a gate is enabled, then the new feature is active, else the old code path is triggered.

With the GateContext that can understand regions, markets, etc, it will be possible to rollout a new functionality to a single region only, then quickly turn it off if it does not work, or release it to all regions.

Reply:
Added to the docs/README file